addons/payment/models/payment_acquirer.py

```diff
@@ -684,6 +684,10 @@ def form_feedback(self, data, acquirer_name):
 
         return True
 
+    @api.model
+    def form_confirmation(self, invoice_id=False):
```

Comment:
I would set this method as private and probably name it to something like _post_process_after_done, because even if it is currently a hack, maybe we will expand this method to correctly handle post payment processing.